### DISCUSSION TITLE
Where is the GUI

### DISCUSSION THREAD

**Discussion by ooshnoo** [grahampugh/erase-install#376] [General]
Testing this out so perhaps i'm dumb, but running the command:  sudo /Library/Management/erase-install/erase-install.sh doesn't show me any GUI windows.  All I see is the whole process going through the terminal window.  

What am I missing here?

**Comment by grahampugh** [comment]
Please check the wiki for the arguments you might want to supply. 

If you run the script without any parameters, you are just downloading the latest installer which happens without any dialogs by default. 

The --erase and --reinstall options have dialogs by default, however.

**Reply by ooshnoo** [reply]
See.. told you I was dumb.  I've been reading the wiki for an hour but didn't see any explicit verbiage about how --reinstall option was required for the GUI.  

Thanks!

**Reply by grahampugh** [reply]
Actually there is a poorly documented option to have a dialog even on the download mode: 

```
--dialog-on-download
```

Looks like I didn't add it to `--help`. I'll add it in for the next version, and put it in the wiki.

**Reply by grahampugh** [reply]
Here you go: https://github.com/grahampugh/erase-install/wiki/3.-Downloading-a-macOS-Installer#--dialog-on-download

**Reply by ooshnoo** [reply]
perfect. Thank you!  this script is great!
